tests/components/material.test.js

```diff
@@ -68,6 +68,15 @@ suite('material', function () {
       assert.notOk(el.sceneEl.materials[oldMaterial.uuid]);
       assert.equal(el.sceneEl.materials[newMaterial.uuid], newMaterial);
     });
+
+    test('texture emits an event on finished loading', function (done) {
+      var el = this.el;
+      el.setAttribute('material', 'src:"test.jpg"');
```

Comment:
adding a true image won't work unless you define the path in the karma config to serve it. we have an example there of loading a true sound file